packages/cli/src/cli/github/make.cog.pr.ts

```diff
@@ -49,7 +49,7 @@ export class MakeCogGithub extends Github {
     this.getBranch(branch);
 
     this.logger.info({ imagery: this.imagery }, 'GitHub: Get the master TileSet config file');
-    if (disabled) {
+    if (addToAerial || category === Category.Event || category === Category.Other) {
```

Comment:
this is inverted? `if (addToAreal) { proceeds to not add to aerial }` ?

Reply:
Good spot. Just fixed.